### DISCUSSION TITLE
[Question] Are there any documents for implementing Hierarchical Reinforcement Learning in Isaac Lab?

### DISCUSSION THREAD

**Discussion by H-Hisamichi** [isaac-sim/IsaacLab#1491] [Q&A] [answered]
Hello everyone,

I'm learning two Hexapod locomotion-Policy. (Walking-Policy & Rolling-Policy)
I also plan to train a high-level policy to select the optimal locomotion mode based on terrain roughness.
Are there any demo codes or resources in Isaac Lab that could help with this?

Thank you!

**Comment by kellyguo11** [comment, marked as answer]
Hello, you can take a look at the navigation example: https://github.com/isaac-sim/IsaacLab/tree/main/source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/navigation

**Reply by H-Hisamichi** [reply]
Hello @kellyguo11,

Thank you for the explanation.
My project uses a direct workflow, so I’ll modify the code to fit my needs. If there are any points to be aware of when connecting direct workflow code with manager-based workflow code, please let me know.
